### DISCUSSION TITLE
Shlink Web Client using wrong health rest endpoint URL

### DISCUSSION THREAD

**Discussion by ARogovskyy** [shlinkio/discussions#30] [General]
#### Shlink web client version

* Version: 3.8.1
* How do you use shlink-web-client: app.shlink.io

#### Summary
Shlink Web Client calls `/rest/v[...]/health` to check for health, but according to the [documentation](https://api-spec.shlink.io/#/Monitoring/health), it should try `/rest/health`. Due to this, app.shlink.io cannot connect to the shlink server.

`rest/v[...]/health` returns 404, and the connection subsequently fails.

**Comment by acelaya** [comment]
It is working as intended. The health endpoint works both with and without version. If it fails it's probably because of some server configuration, reverse proxy, etc.
